corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -87,20 +87,21 @@ EntrySingleAnswer = function(question, options) {
     var self = this;
 
     Entry.call(self, question, options);
-    var extensions = {};
-    var displayOptions = _getDisplayOptions(question)
     self.valueUpdate = undefined;
-    if (options.enableRateLimit && ko.utils.unwrapObservable(displayOptions.phoneMode)) {
-        extensions.rateLimit = {
-            timeout: Formplayer.Const.KO_ENTRY_TIMEOUT,
-            method: "notifyWhenChangesStop"
-        };
-        self.valueUpdate = 'keyup';
-    }
-    self.rawAnswer = ko.observable(question.answer() || Formplayer.Const.NO_ANSWER)
-        .extend(extensions);
+    self.rawAnswer = ko.observable(question.answer() || Formplayer.Const.NO_ANSWER);
 
     self.rawAnswer.subscribe(self.onPreProcess.bind(self));
+
+    var displayOptions = _getDisplayOptions(question);
+    if (options.enableAutoUpdate && ko.utils.unwrapObservable(displayOptions.phoneMode)) {
```

Comment:
one thing that make make this cleaner, is to only have `enabledAutoUpdate` be `true` if `phoneMode` is true when we pass in the option to the control